src/ScheduleFunctions.cpp

```diff
@@ -747,12 +732,11 @@ class IsUsedInStmt : public IRVisitor {
 
 public:
     bool result;
-    IsUsedInStmt(Function f) : func(f.name()), result(false) {}
-
+    explicit IsUsedInStmt(const Function &f) : func(f.name()), result(false) {}
```

Comment:
the func member can be a const string & to skip the string copy (f.name() returns a const ref).

Reply:
It can